ReText/editor.py

```diff
@@ -246,6 +249,10 @@ def keyPressEvent(self, event):
 					# Insert Markdown-style line break
 					cursor.insertText('  ')
 				self.handleReturn(cursor)
+		elif key == Qt.Key_Down and event.modifiers() & Qt.AltModifier:
+			self.moveLineDown()
+		elif key == Qt.Key_Up and event.modifiers() & Qt.AltModifier:
+			self.moveLineUp()
```

Comment:
These lines are no longer needed, the shortcut is handled by QAction.

Reply:
done